04_mini-projects/00_ode.ipynb

```diff
@@ -426,14 +430,14 @@
     "    y[0] = inc\n",
     "    \n",
     "    for idx, t in enumerate(time_vec[1:]):\n",
-    "        f = func(t, np.array([y[idx]]))[0]\n",
+    "        f = func(t, np.array([y[idx]]))\n",
```

Comment:
The actual change is here, all other lines are just due to reset of cell execution counts.